Directory.Build.props

```diff
@@ -1,4 +1,8 @@
 <Project>
+  <Import
+    Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))\AspNetCoreSettings.props"
```

Comment:
👍 

Should we also add this to repo.props? I imagine there will be other properties that we need in the korebuild context, not just csproj build context

Reply:
Yup, that was my intention here.